ghost/posts-service/lib/PostsService.js

```diff
@@ -61,6 +62,46 @@ class PostsService {
             }
         }
 
+        if (this.isSet('collections') && frame.data.posts[0].collections) {
```

Comment:
Yup! This is pretty much the the way to do it imo. It's super verbose, but whatever. My understanding is that we are allowing to use an array of string ids or an array of objects with ids here. It would follow the same pattern we have for tags/labels, but for those resources we also allow assigning by "slug". I'm a little on the fence if we should mimic existing pattern 1:1 OR support even less syntax options at least in early stages. Not a big deal which way we go with this or stay with current approach. What is your thinking on this topic?